lib/constants.js

```diff
@@ -1,33 +1,39 @@
-const ORD_RESOURCE_TYPE = Object.freeze(
-    {
-        "service": "service",
-        "entity": "entity",
-        "event": "event",
-        "api": "api"
-    });
-
-const COMPILER_TYPES = Object.freeze(
-    {
-        "oas3": "oas3",
-        "asyncapi2": "asyncapi2",
-        "edmx": "edmx"
-    });
+const COMPILER_TYPES = Object.freeze({
+    oas3: "oas3",
+    asyncapi2: "asyncapi2",
+    edmx: "edmx",
+});
 
 const CONTENT_MERGE_KEY = "ordId";
 
-const ORD_EXTENSIONS_PREFIX = "@ORD.Extensions.";
+const DESCRIPTION_PREFIX = "Description for ";
 
 const OPEN_RESOURCE_DISCOVERY_VERSION = "1.9";
 
+const ORD_EXTENSIONS_PREFIX = "@ORD.Extensions.";
+
+const ORD_RESOURCE_TYPE = Object.freeze({
+    service: "service",
+    entity: "entity",
+    event: "event",
+    api: "api",
+});
+
+const RESOURCE_VISIBILITY = Object.freeze({
```

Comment:
@zongqichen : Minor change - just sorted the constants alphabetically. This is recommended by the linter.

Reply:
How do you do it? I like it, in goland, there is a linter can achieve. Is it an extension in vscode which we could auto-save? We should put it into vscode settings, which any developer can follow same way

Reply:
It is configuration in linter which is not part of the plugin but set in calesi-root.
This time I did it by using GH-Copilot. I find alphabetical sorting `require` and `constants` standing in the header pretty convenient. We can do that in a separate PR.